biz.aQute.repository/src/aQute/bnd/repository/maven/provider/IndexFile.java

```diff
@@ -64,7 +68,11 @@
  * results without having to wait for the operation to be done.
  */
 class IndexFile {
-	private final static Logger					logger		= LoggerFactory.getLogger(IndexFile.class);
+	private static final String[]				ZIP_EXTENSIONS	= new String[] {
```

Comment:
You may wish to use the new `Sets.of` method to create an immutable set of the extensions?